utils/rrdbackup/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rrdbackup
+PKG_VERSION:=1
+PKG_RELEASE:=1
+
+PKG_LICENSE:=Unlicense
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/rrdbackup
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Enable the backup and restore of the RRD database for collectd during orderly reboots
+  URL:=https://github.com/sqrwf/openwrt-rrdbackup
+  PKGARCH:=all
+  MAINTAINER:=John Kohl <jtk.git@bostonpog.org>
+  EXTRA_DEPENDS:=collectd-mod-rrdtool luci-app-statistics
```

Comment:
The script depends on neither (those two packages are for visual output in LuCI), it actually doesn't depend on anything really. You can of course make the case that it doesn't make much sense without `collectd` itself.

Reply:
so what are the "right" packages to depend upon?  I'd want it to depend on the packages that generate files in /tmp/rrd.  That's not strictly collectd itself, from what I have examined.

The writing to /tmp/rrd comes from the rrdtool plugin into collectd, and the configuration file generated to use that plugin.
I see that /etc/init.d/luci_statistics generates /var/etc/collectd.conf which refers to the rrdtool plugin and configures it to store files in /tmp/rrd, from where rrdbackup saves/restores the files.

So then really I guess the dependency is best expressed on /etc/init.d/luci_statistics (which generates the configuration for collectd to use /tmp/rrd), so I'd say it should depend on luci-app-statistics.
And then what controls whether luci_statistics generates an output to /tmp/rrd ?  I'd say that is when it's configured to use the rrd output plugin, which is I think collectd-mod-rrdtool...but  the .js file with the configuration panel for rrdtool is in luci-app-statistics, which itself depends on collectd-mod-rrdtool, so again this points to luci-app-statistics.

And, this script only works correctly if /tmp/rrd is configured as the pathname in /etc/config/luci_statistics...so maybe there's a change needed to get that name through something like `$(uci get luci_statistics.collectd_rrdtool.DataDir)`
?

and the depends can be trimmed to just luci-app-statistics